src/state.rs

```diff
@@ -262,6 +262,11 @@ impl StateSet {
         self.0.contains(other)
     }
 
+		/// Removes a [`State`] (optionally) previously contained in the [`StateSet`].
```

Comment:
I don't like having links pointing to the current item inside its documentation. You already know you are reading the documentation for `StateSet`. Therefore it's only noise, especially for screen reader users.